src/gl.rs

```diff
@@ -275,6 +285,59 @@ pub fn tex_image_2d(target: GLenum,
 }
 
 // FIXME: Does not verify buffer size -- unsafe!
+pub fn compressed_tex_image_2d(target: GLenum,
+                               level: GLint,
+                               internal_format: GLenum,
+                               width: GLsizei,
+                               height: GLsizei,
+                               border: GLint,
+                               image_size: GLsizei,
+                               opt_data: Option<&[u8]>) {
```

Comment:
Just to verify: does `CompressedTexImage2D` work the same way as `glTexImage2D` in that a null `data` argument allocates a buffer without copying to it?


If so, could you add doc comments on these two methods explaining how `opt_data` works?

Thanks!